tests/connect/standalone-ssl-no_verify-003.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+Connect to MongoDB with SSL and no host/cert verification (URI options)
```

Comment:
I don't believe we had any existing tests for the non-deprecated URI options, so it made sense to add copies of these three tests. The expectations are identical apart from the absent deprecation notices.